src/core/storage/storagekeys.js

```diff
@@ -6,18 +6,21 @@
  *
  * @enum {string}
  */
-export default {
+const StorageKeys = {
   NAVIGATION: 'navigation',
   UNIVERSAL_RESULTS: 'universal-results',
   VERTICAL_RESULTS: 'vertical-results',
   ALTERNATIVE_VERTICALS: 'alternative-verticals',
   AUTOCOMPLETE: 'autocomplete',
   DIRECT_ANSWER: 'direct-answer',
   FILTER: 'filter', // DEPRECATED
-  STATIC_FILTER_NODE: 'static-filter-node',
+  PERSISTED_FILTER: 'filters',
+  STATIC_FILTER_NODES: 'static-filter-nodes',
+  LOCATION_RADIUS_FILTER_NODE: 'location-radius-filter-node',
+  PERSISTED_LOCATION_RADIUS: 'locationRadius',
```

Comment:
it's a bit weird to use `locationRadius` when we have `location-radius-filter-node`. is this because we want it to match some LiveAPI format?

Reply:
yup, locationRadius is just a number like 1000.5. location-radius-filter-node is the whole node with unneeded metadata